CHANGELOG.md

```diff
@@ -70,6 +71,7 @@ To learn more about active deprecations, we recommend checking [GitHub Discussio
 
 - **General:** Provide patch for CVE-2022-3172 vulnerability ([#3690](https://github.com/kedacore/keda/issues/3690))
 - **General:** Respect optional parameter inside envs for ScaledJobs ([#3568](https://github.com/kedacore/keda/issues/3568))
+- **General:** A problem with PushScaler ([#3881](https://github.com/kedacore/keda/issues/3881))
```

Comment:
Let's do this in the PR that fixes this problem
```suggestion
```

Reply:
There is [another PR](https://github.com/kedacore/keda/pull/3599) for that indeed

Reply:
all done